src/WebJobs.Extensions.DurableTask/ContextImplementations/DurableCommonContext.cs

```diff
@@ -250,7 +255,7 @@ internal virtual void ThrowIfInvalidAccess()
                     break;
 
                 case FunctionType.Entity:
-                    System.Diagnostics.Debug.Assert(!string.IsNullOrEmpty(operation), "The operation parameter is required.");
+                    System.Diagnostics.Debug.Assert(operation != null, "The operation parameter is required.");
```

Comment:
Did something change to allow this to be `""` now?

Reply:
Looks like you answered this above.

Reply:
Just that the REST API made me rethink the original design. I don't see a strong reason why we need to insist on an operation name being specified.